### PR TITLE
:window: :tada: Add integration for Osano/GDPR

### PR DESCRIPTION
**Pull request by timroes**
## What

Closing https://github.com/airbytehq/airbyte-cloud/issues/2426

This adds the full implementation for GDPR and Osano into the product. It will address the following:

* Loads the Osano script/styles only in Cloud now (as per discussion with Andy)
* Add a "Cookie Preferences" button in the sidebar in Cloud settings that will open the Osano drawer. This button will only be shown if Osano is switched to "active" (i.e. `production`) and thus show the consent dialogs.
* For users in GDRP affected countries make the newsletter an opt-in instead of opt-out during sign-up

### CODE REVIEW

**Review by krishnaglick — APPROVED** [review]
Did not test locally, code LGTM

**Review by joeykmh — APPROVED** [review]
Also did not test locally, but looks good to me 👍

File: airbyte-webapp/src/utils/dataPrivacy.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  "Europe/Riga",
+  "Europe/Rome",
+  "Europe/Sofia",
+  "Europe/Stockholm",
+  "Europe/Tallinn",
+  "Europe/Vienna",
+  "Europe/Vilnius",
+  "Europe/Warsaw",
+  "Europe/Zagreb",
+];
```
Comment (josephkmh) [inline airbyte-webapp/src/utils/dataPrivacy.ts:46]: Just curious: is browser timezone a sufficient way to enforce GDPR? Seems a bit unreliable to me, but I don't have a better suggestion.

Comment (timroes) [reply airbyte-webapp/src/utils/dataPrivacy.ts:46]: I agree, this is not a really foolproof way. Luckily this is really only used to determine the default state of the newsletter flag and such I think it's an "okay enough" approach, without requiring to get a third party geo location service in place (which would be the better approach detecting by IP location). Osano itself for showing the right consent banner actually uses a proper IP based approach, so it's really only for the default state of this one switch we're needing to use this weaker approach.

**Pull request merged**